CoreFramework/NUnitTestSetup/NUnitTestSetup.cs

```diff
@@ -6,15 +6,13 @@ namespace CoreFramework.NUnitTestSetup;
 public class NUnitTestSetup
 {
     protected WebDriverAction? DriverBaseAction;
-    private string Author = "Hong_Anh_Pham";
     private string Device = "PC";
     private string Category = "Phase2_TestProject";
     [OneTimeSetUp]
     public void OneTimeSetUp()
     {
         HtmlReport.CreateReport();
-        HtmlReport.CreateTest(TestContext.CurrentContext.Test.ClassName).
-            AssignAuthor(Author).AssignDevice(Device).AssignCategory(Category);
+        HtmlReport.CreateTest(TestContext.CurrentContext.Test.ClassName).AssignDevice(Device).AssignCategory(Category);
```

Comment:
@honganh1206 This is a better choice for HTML test defined

Reply:
Got it! Will fix this asap, thanks!